put_message_on_bucket.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import json
+
+from google.cloud import pubsub_v1, storage
+from termcolor import colored
+
+
+def main(subscription_name, project_id, number_messages, id_search, action, bucket_name):
+    subscriber = pubsub_v1.SubscriberClient()
+    response = []
+
+    subscription_path = subscriber.subscription_path(
+        project_id, subscription_name)
+    for _ in range(number_messages):
+        message = subscriber.pull(subscription_path, max_messages=1, return_immediately=True)
+        if not message.received_messages:
+            break
+        response.extend(message.received_messages)
+    if not response:
+        print(colored('No messages on pubsub ', 'red'))
+        return
+    elif id_search:
+        matching_id_messages = [msg for msg in response if id_search == msg.message.message_id]
+        if matching_id_messages:
+            if action == 'MOVE':
```

Comment:
Might be able to do away with needing the `MOVE` action now?

Reply:
I kept the `MOVE` in there just to be explicit on what you're doing with the message. Happy to remove it if you think I should

Reply:
I guess we could make `MOVE` a mandatory argument when the script is called, or could rename the script to `move_message_to_bucket.py` or something similar? Happy to go with what you reckon is best 👍